core/src/main/java/com/linecorp/armeria/client/AbstractHttpRequestHandler.java

```diff
@@ -202,6 +204,16 @@ final void writeHeaders(RequestHeaders headers) {
         final RequestHeaders merged = mergeRequestHeaders(
                 headers, ctx.defaultRequestHeaders(), ctx.additionalRequestHeaders());
         logBuilder.requestHeaders(merged);
+
+        if (headers.contains(HttpHeaderNames.CONNECTION, HttpHeaderValues.CLOSE.toString())) {
```

Comment:
Could we extract `HttpHeaderValues.CLOSE.toString()` to a `private static final` field? `toString()` is cheap but it may not be free.